### PR TITLE
Fix/update abis and refactor bridge abi management

### PR DESCRIPTION
**Pull request by IOVgomezdn**

### CODE REVIEW

**Review by sergio-jara — APPROVED** [review]
I choose to believe

File: src/lib/nativeContracts/bridgeAbi.js [hunk 1]
```diff
@@ -1,2 +1,47 @@
...
+    { height: 4015800, abi: fingerroot }
+  ]
+}
+
+function findmatchingAbi (txHeight, abisWithHeight) {
+  const lastIndex = abisWithHeight.length - 1
+
+  if (txHeight >= abisWithHeight[lastIndex].height || txHeight === undefined) {
+    return abisWithHeight[lastIndex].abi
+  } else {
```
Comment (ilanolkies) [inline src/lib/nativeContracts/bridgeAbi.js]: I would remove else after return

See https://eslint.org/docs/latest/rules/no-else-return#rule-details

**Review by ilanolkies — CHANGES_REQUESTED** [review]
👌🏻 good testing

File: src/lib/nativeContracts/bridgeAbi.js (hunk 1 above)
Comment (IOVgomezdn) [reply src/lib/nativeContracts/bridgeAbi.js]: done

**Review by ilanolkies — APPROVED** [review]

**Pull request merged**